### PR TITLE
docs: [SIW-1136] Update README.md

### PR DESCRIPTION
**Pull request by LazyAfternoons**
### Short description
This PR updates the main repository README.md for both Android and iOS implementations.

#### How Has This Been Tested?
Check the [README.md](https://github.com/pagopa/io-react-native-integrity/blob/f2751769953052a615d4743089b49cf447635529/README.md) preview.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,184 @@
-# @pagopa/io-react-native-integrity
+# io-react-native-integrity
```
Comment (hevelius) [inline README.md]: Why removing @pagopa?

File: README.md [hunk 2]
```diff
@@ -1,23 +1,184 @@
...
 
-React Native package which bridges Andoird Play Integrity API and iOS App Attest
+React Native interfaces for managing secure storage in iOS and Android.
 
 ## Installation
 
 ```sh
-npm install @pagopa/io-react-native-integrity
+npm install io-react-native-integrity
```
Comment (hevelius) [inline README.md]: Trying to install `io-react-native-integrity` returns an error
`npm ERR! 404 Not Found - GET https://registry.yarnpkg.com/io-react-native-integrity - Not found
`

File: README.md [hunk 3]
```diff
@@ -1,23 +1,184 @@
...
+|          GENERATION_KEY_FAILED           |     iOS     | A critical error occurred during the `generateHardwareKey` operation                                 |
+|            ATTESTATION_ERROR             |     iOS     | A critical error occurred during the `getAttestation` operation                                      |
+|         UNSUPPORTED_IOS_VERSION          |     iOS     | The device has a version of iOS that doesn't support the DeviceCheck App Attestation Service (<14.0) |
+|             CHALLENGE_ERROR              |     iOS     | An error occured while encoding the provided challenge to `getAttestation`                           |
+|        CLIENT_DATA_ENCODING_ERROR        |     iOS     | An error occured while encoding the provided client data to `generateHardwareSignatureWithAssertion` |
+|       GENERATION_ASSERTION_FAILED        |     iOS     | A critical error occurred during the `generateHardwareSignatureWithAssertion` operation              |
+|            UNSUPPORTED_DEVICE            | iOS/Android | The device doesn't support the requested functionality                                               |
+
```
Comment (hevelius) [inline README.md:185]: kudos, very usefully

**Review by hevelius — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hevelius) [reply README.md]: Works well with `@pagopa/io-react-native-integrity`

File: README.md (hunk 1 above)
Comment (LazyAfternoons) [reply README.md]: No idea why that got removed honestly. Thanks.

File: README.md (hunk 2 above)
Comment (LazyAfternoons) [reply README.md]: Fixed in fa3ecb5df3419bd1001f1e20952ef71d3c12ba2a, thanks.

File: README.md
Comment (hevelius) [inline README.md:1]: What do you think to add also status badge like:

CI workflow
Publish workflow
Code QL workflow

**Review by hevelius — CHANGES_REQUESTED** [review]

File: README.md
Comment (LazyAfternoons) [reply README.md:1]: Done in f2751769953052a615d4743089b49cf447635529, thanks.

**Review by hevelius — APPROVED** [review]
LGTM

**Pull request merged**